cwms_radar_api/src/main/java/cwms/radar/api/TimeSeriesController.java

```diff
@@ -221,31 +221,28 @@ public void getAll(Context ctx) {
                 DSLContext dsl = getDslContext(ctx))
         {
             TimeSeriesDao dao = getTimeSeriesDao(dsl);
-            String format = ctx.queryParamAsClass("format",String.class).getOrDefault("");
+            String format = ctx.queryParamAsClass("format", String.class).getOrDefault("");
             String names = ctx.queryParam("name");
             String office = ctx.queryParam("office");
-            String unit = ctx.queryParamAsClass("unit",String.class).getOrDefault(UnitSystem.EN.getValue());
+            String unit = ctx.queryParamAsClass("unit", String.class).getOrDefault(UnitSystem.EN.getValue());
             String datum = ctx.queryParam("datum");
             String begin = ctx.queryParam("begin");
             String end = ctx.queryParam("end");
             String timezone = ctx.queryParam("timezone");
             // The following parameters are only used for jsonv2 and xmlv2
-            String cursor = ctx.queryParamAsClass("cursor",String.class)
-                                .getOrDefault(ctx.queryParamAsClass("page",String.class)
-                                .getOrDefault("")
-                                );
+            String cursor = ctx.queryParamAsClass("cursor", String.class).getOrDefault(
+                    ctx.queryParamAsClass("page", String.class).getOrDefault(""));
 
-            int pageSize = ctx.queryParamAsClass("pageSize",Integer.class)
-								.getOrDefault(
-									ctx.queryParamAsClass("pagesize",Integer.class).getOrDefault(defaultPageSize)
-								);
+            int pageSize = ctx.queryParamAsClass("pageSize", Integer.class).getOrDefault(
+                    ctx.queryParamAsClass("pagesize", Integer.class).getOrDefault(defaultPageSize));
 
             String acceptHeader = ctx.header(Header.ACCEPT);
             ContentType contentType = Formats.parseHeaderAndQueryParm(acceptHeader, format);
 
             String results;
             String version = contentType.getParameters().get("version");
-            if(version != null && version.equals("2")) {
+            if(version != null && version.equals("2"))
+            {
```

Comment:
Does our checkstyle file indicate { on the next line or is your IDE set for a different project?
(I'm not a die hard for either form it just seems odd that your more recent PRs have starting doing this.)

Reply:
I updated an ide plugin.  Must have lost the settings.  I'll look into it.